provider/objectstorage_preauthrequest_resource.go

```diff
@@ -3,11 +3,11 @@
 package provider
 
 import (
-	"github.com/hashicorp/terraform/helper/schema"
-	"github.com/oracle/bmcs-go-sdk"
-
 	"time"
 
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/oracle/bmcs-go-sdk"
 	"github.com/pkg/errors"
 
```

Comment:
nit: Can you remove those empty lines between the github imports while you're at it :) I know it's not yours, but I saw them in a couple of places.

Reply:
sure .. if i have to make another commit, I will take that ?

Reply:
make fmt (goimports) will add space between standard go libs and 3rd party stuff, so we shouldn't fight it. It does fail at proper reformatting if there are odd new lines and custom import placement. If you want to see what it's supposed to look like according to goimports, collapse all empty lines and see what make fmt does

Reply:
I will keep an eye on this. Since Brian has approved this I'll go ahead and merge this.

Thanks guys.